lib/utils.pm

```diff
@@ -1313,6 +1314,16 @@ sub snapper_revert_system {
     select_console 'root-console';
 }
 
+=head2 get_root_console_tty
+    Returns tty number used designed to be used for root-console.
+    When console is not yet initialized, we cannot get it from arguments.
+    Since SLE 15 gdm is running on tty2, so we change behaviour for it and
+    openSUSE distris.
+=cut
+sub get_root_console_tty {
+    return (sle_version_at_least('15') && !is_caasp) ? 6 : 2;
+}
+
```

Comment:
`lib/utils.pm` is getting quite big. We should really try to find more specific locations, base classes, etc.